src/wp-includes/class-wp-term-query.php

```diff
@@ -160,6 +160,8 @@ class WP_Term_Query {
 	 *                                                   (even if `$hide_empty` is set to true). Default true.
 	 *     @type string          $search                 Search criteria to match terms. Will be SQL-formatted with
 	 *                                                   wildcards before and after. Default empty.
+	 *     @type string          $s                      Alias of `$search`. Default empty.
+	 *                                                   @since 6.7.0
```

Comment:
It should be added in line no. 95.

`@since 6.7.0 Introduced the 's' parameter.`

Reply:
Made the requested changes.